commands/commands.go

```diff
@@ -0,0 +1,186 @@
+	RootCmd.PersistentFlags().BoolVar(&preCfg.QTLSSkipVerify, "qtlsskipverify", fileCfg.QTLSSkipVerify, "skip TLS verification")
+
+	RootCmd.PersistentFlags().Int64Var(&preCfg.Confirmations, "confirmations", 10, "Number of block confirmations ")
+	RootCmd.PersistentFlags().BoolVar(&preCfg.UI, "ui", true, "Start Wallet with RPC and webUI interface")
+	RootCmd.PersistentFlags().StringArrayVar(&preCfg.Listeners, "listeners", fileCfg.Listeners, "rpc listens")
+	RootCmd.PersistentFlags().StringVar(&preCfg.RPCUser, "rpcUser", fileCfg.RPCUser, "rpc user,default by random")
+	RootCmd.PersistentFlags().StringVar(&preCfg.RPCPass, "rpcPass", fileCfg.RPCPass, "rpc pass,default by random")
+	RootCmd.PersistentFlags().Int64Var(&preCfg.MinTxFee, "mintxfee", fileCfg.MinTxFee, "The minimum transaction fee in QIT/kB default 20000 (aka. 0.0002 MEER/KB)")
+}
+
+// LoadConfig config file and flags
+func LoadConfig(cmd *cobra.Command, args []string) {
+	// load configfile ane merge command ,but don't udpate configfile
+	_, err := toml.DecodeFile(preCfg.ConfigFile, fileCfg)
+	if err != nil {
+		if cmd.Flag("configfile").Changed {
+			if fExit, _ := utils.FileExists(preCfg.ConfigFile); fExit {
+				log.Error(fmt.Sprintf("config file err: %s", err))
+				return
+			}
+			return
+		}
+	}
+
+	fileCfg.ConfigFile = preCfg.ConfigFile
+
+	if cmd.Flag("debuglevel").Changed {
+		fileCfg.DebugLevel = preCfg.DebugLevel
+	}
+	if cmd.Flag("appdatadir").Changed {
+		fileCfg.AppDataDir = preCfg.AppDataDir
+	}
+	if cmd.Flag("logdir").Changed {
+		fileCfg.LogDir = preCfg.LogDir
+	}
+	if cmd.Flag("network").Changed {
+		fileCfg.Network = preCfg.Network
+	}
+	if cmd.Flag("create").Changed {
+		fileCfg.Create = preCfg.Create
+	}
+	if cmd.Flag("listeners").Changed {
+		fileCfg.Listeners = preCfg.Listeners
+	}
+	if cmd.Flag("rpcUser").Changed {
+		fileCfg.RPCUser = preCfg.RPCUser
+	}
+	if cmd.Flag("rpcPass").Changed {
+		fileCfg.RPCPass = preCfg.RPCPass
+	}
+	if cmd.Flag("ui").Changed {
+		fileCfg.UI = preCfg.UI
+	}
+	if cmd.Flag("qserver").Changed {
+		fileCfg.QServer = preCfg.QServer
+	}
+	if cmd.Flag("quser").Changed {
+		fileCfg.QUser = preCfg.QUser
+	}
+	if cmd.Flag("qpass").Changed {
+		fileCfg.QPass = preCfg.QPass
+	}
+	if cmd.Flag("pubwalletpass").Changed {
+		fileCfg.WalletPass = preCfg.WalletPass
+	}
+	if cmd.Flag("qnotls").Changed {
+		fileCfg.QNoTLS = preCfg.QNoTLS
+	}
+	if cmd.Flag("qcert").Changed {
+		fileCfg.QCert = preCfg.QCert
+	}
+	if cmd.Flag("mintxfee").Changed {
+		fileCfg.MinTxFee = preCfg.MinTxFee
+	}
+	if cmd.Flag("confirmations").Changed {
+		fileCfg.Confirmations = preCfg.Confirmations
+	}
+	if cmd.Flag("qtlsskipverify").Changed {
+		fileCfg.QTLSSkipVerify = preCfg.QTLSSkipVerify
+	}
+	config.ActiveNet = utils.GetNetParams(fileCfg.Network)
+	funcName := "LoadConfig"
+	// Parse, validate, and set debug log level(s).
+	if err := parseAndSetDebugLevels(fileCfg.DebugLevel); err != nil {
+		err := fmt.Errorf("%s: %v", funcName, err.Error())
```

Comment:
`funcName` looks only used here.

Reply:
this function will be substituted by viper, #43